crates/node/core/src/args/pruning.rs

```diff
@@ -3,48 +3,248 @@
 use clap::Args;
 use reth_chainspec::ChainSpec;
 use reth_config::config::PruneConfig;
+use reth_primitives::{Address, BlockNumber};
 use reth_prune_types::{PruneMode, PruneModes, ReceiptsLogPruneConfig, MINIMUM_PRUNING_DISTANCE};
+use std::collections::BTreeMap;
 
 /// Parameters for pruning and full node
 #[derive(Debug, Clone, Args, PartialEq, Eq, Default)]
 #[command(next_help_heading = "Pruning")]
 pub struct PruningArgs {
     /// Run full node. Only the most recent [`MINIMUM_PRUNING_DISTANCE`] block states are stored.
-    /// This flag takes priority over pruning configuration in reth.toml.
```

Comment:
good call to remove, this is not true atm, maybe it will be enforced later, but for now, you can add "Closes https://github.com/paradigmxyz/reth/issues/9808" to the PR description